### PR TITLE
Port tvOS headers

### PR DESCRIPTION
**Pull request by imurchie**
Begin porting tvOS support into Appium fork. This covers https://github.com/facebook/WebDriverAgent/pull/1032

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Used for HTTP routing
-github "marekcirkos/RoutingHTTPServer"
+github "shvul/RoutingHTTPServer"
```
Comment (mykola-mokhnach) [inline Cartfile]: I'd rather move this component to appium org

Comment (imurchie) [reply Cartfile]: Me too. He has a fork of the Cocoa HTTP server too.

**Review by mykola-mokhnach — APPROVED** [review]

**Review by KazuCocoa — APPROVED** [review]

**Review by jlipps — APPROVED** [review]

**Comment by KazuCocoa** [comment]
will we merge this? 👀

**Comment by mykola-mokhnach** [comment]
@imurchie I think these fixes to tests might look better if moved to a separate PR

**Comment by jlipps** [comment]
i'm interested in this too, any more ideas @imurchie?

**Comment by KazuCocoa** [comment]
https://github.com/appium/WebDriverAgent/pull/148
https://github.com/appium/WebDriverAgent/pull/149
https://github.com/appium/WebDriverAgent/pull/151

**Pull request closed without merging**